.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
what does this do?

Reply:
I think it just automatically set when I set my typescript language settings locally.